### DISCUSSION TITLE
Is there a way to JOIN sequence from stream with the sequence from lookup table by some row index?

### DISCUSSION THREAD

**Discussion by vol-ale** [lf-edge/ekuiper#2133] [Q&A]
Let's assume I have one sequence of doubles stored in the lookup table and another one obtained from the stream using counting window. Is there a way to JOIN such sequences by some row index? For example, I need to compute the Euclidean distance between these sequences.

**Comment by ngjaying** [comment]
It should be supported. It may not be heavily tested for this scenario.

**Reply by vol-ale** [reply]
@ngjaying,  
Can you please provide an example? I haven't found any analogue of RowID, ROW_NUMBER, etc. in documentation.

**Reply by ngjaying** [reply]
Ah, there is no row_number now.

**Reply by vol-ale** [reply]
Any plans to add it soon?

**Reply by ngjaying** [reply]
You can open an issue for the feature.  Maybe with a more detail scenario description. We will prioritize. Thanks.
